compute/admin_guide/agentless-scanning/onboard-accounts/configure-gcp.adoc

```diff
@@ -0,0 +1,93 @@
+
+[.task]
+=== Configure Agentless Scanning for GCP Accounts
+
+[.procedure]
+
+. Log in to the Prisma Cloud administrative console.
+
+. Select *Compute > Manage > Cloud Accounts*.
+
+. Click the edit button of your cloud account.
+
+. Go to the *Agentless Scanning* section.
+
+. Expand the *Advanced settings* and provide the following information.
++
+image::agentless-gcp-pcee-advanced-settings.png[width=540]
+
+.. Enable Permissions check to verify that the permissions are correct before running a
+scan.
+.. *Scanning type*: For GCP accounts, you can decide between xref:../agentless-scanning.adoc#scanning-modes[two scanning modes].
+
+... *Same Account*: Scan hosts of your GCP account using that same account. Use this value for the account you want to use as the hub account.
+
+... *Hub Account*: Scan hosts of your GCP account using a different account. Select another onboarded account from the list to scan the account you are configuring.
+
+.. Enter a *Proxy* value if traffic leaving your GCP tenant uses a proxy.
+
+.. Under *Scan scope* you can refine the scope of the scanning by *Regions* or using tags.
++
+image::tags-scope.png[width=300]
+
+... Choose *All regions* to scan in all GCP regions.
+If you choose *Custom regions*, enter the GCP region in which you want Prisma Cloud to scan.
+
+... Choose whether to *Exclude hosts by tags* or to *Include hosts by tags*.
+You can use wildcards to refine the scanning as needed.
+
+.. Choose whether or not to *Scan non running hosts*.
+
+.. Choose whether or not to enable *Auto-scale scanning*. If you disable auto-scale, specify number of
+scanners Prisma Cloud should employ.
+
+. Click Next.
+
+. Leave the *Discovery features* unchanged.
+
+. Click *Save* to return to *Compute > Manage > Cloud accounts*.
+
+==== Use Encrypted Volumes
+
+When you use xref:../agentless-scanning-modes.adoc[hub and target projects], you can configure your hub project to access the encrypted volumes of the target accounts.
+To use encrypted volumes the service account of Google Compute Engine needs to have the `cloudkms.cryptoKeyEncrypterDecrypter` role.
+Without it, the service agent of the the hub project can't access the KMS keys.
+
+The Compute Engine service agent for your hub project is labeled with the following convention.
+`service-PROJECT_NUMBER@compute-system.iam.gserviceaccount.com`
+Replace `PROJECT_NUMBER` with the number of your hub project.
+
+Use the following command to apply the grant the role and permissions to the Compute Engine service agent.
+
+[source]
+----
+gcloud projects add-iam-policy-binding KMS_PROJECT_ID \
+    --member serviceAccount:service-PROJECT_NUMBER@compute-system.iam.gserviceaccount.com \
+    --role roles/cloudkms.cryptoKeyEncrypterDecrypter
+----
+
+Replace `KMS_PROJECT_ID` with any project you need to use.
+The project doesn't need to be the hub or the target projects you wish to scan.
```

Comment:
"It is not required that the KMS project is the hub or the target projects you wish to scan."